src/index.html

```diff
@@ -6,13 +6,67 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        class="product-card__image"
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac"
+      />
+      <div class="product-card__content">
+        <h2 class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="product-card__code">Product code: 195434</p>
+        <div
+          class="product-card__rating"
+          data-qa="hover"
+        >
+          <div class="product-info">
+            <div class="product_stars">
```

Comment:
1. it's a `stars` block
2. add a modifier to specify how may of stars are active
```suggestion
            <div class="stars stars--4">
```

Reply:
так у мене нижче указує скільки активних

          <div class="product_stars">
            <div class="product_stars__star product_stars__star--4"></div>
            <div class="product_stars__star product_stars__star--star"></div>
            <div class="product_stars__star product_stars__star--star"></div>
            <div class="product_stars__star product_stars__star--star"></div>
            <div class="product_stars__star"></div>
          </div>